tests/unit/EntityMenu.spec.ts

```diff
@@ -478,7 +478,9 @@ describe('Entity Menu - Consent to Continuation click tests', () => {
   })
 })
 
-describe('Entity Menu - Request AGM Extension click tests', () => {
+describe.skip('Entity Menu - Request AGM Extension click tests', () => {
```

Comment:
disabled this for now. Looking into it

Reply:
As this ticket is taking longer than expected, please add a specific comment to the next ticket to fix this (these) unit test(s). And then leave this (these) disabled in this PR, so we can move on.

Reply:
Sounds good to me. I think this ticket became a bit bigger because of some dependencies and unexpected changes. I will add comments to next ticket.